### ISSUE TITLE
NN: Loss functions library

### ISSUE THREAD

**Issue by petrucristescu** [petrucristescu/lamc#63]
## Summary
Create `examples/digits/lib/loss.ch` with loss functions for training.

Part of #59. Depends on #60, #62.

## Functions
- `crossEntropy predicted label` — -log(predicted[label])
- `oneHot label size` — target vector [0,...,1,...,0]

## Tests
Integration test in `examples/digits/test/test_loss.ch`

**Comment by petrucristescu** [comment]
Implemented in feat commit.
